filters-api.md

```diff
@@ -1,10 +1,10 @@
-# V2 Filters
+# API Filters (v2)
```

Comment:
Since this isn't strictly an API, maybe we can title it something like "Filter Parameters (v2)"? Thoughts?